src/utils/workflow-utils.ts

```diff
@@ -132,10 +147,14 @@ export const updateFullSource = () => {};
 export const generateTaskSpec = (
     entityType: EntityWithCreateWorkflow,
     connectorConfig: ConnectorConfig,
-    resourceConfigs: ResourceConfigDictionary | null,
+    resourceConfigs: ResourceConfigDictionary,
+    resourceConfigServerUpdateRequired: boolean,
+    bindings: Bindings,
     existingTaskData: DraftSpecsExtQuery_ByCatalogName | null,
-    sourceCapture: string | null,
-    fullSource: FullSourceDictionary | null
+    options: {
+        fullSource: FullSourceDictionary | null;
+        sourceCapture: string | null;
+    }
```

Comment:
These should stay params and not nested in an `options` object.

Reply:
This was a _quick_ way to handle the fact that we have exceeded the maximum number of parameters. Ultimately, I wanted to move this into a hook but felt that may be too large of a change for this PR.